### ISSUE TITLE
Attribute Default Value

### ISSUE THREAD

**Issue by mohameddiaa27** [aaronmallen/activeinteractor#149]
#### Elevator pitch, describe and sell us your feature request

**As a** rails developer,
**I want** to be able to define default value to a specific attribute
**so that** I can have a dry code for default usage

#### Is your feature request related to a problem

I'm always frustrated when I pass variables every time to make the command dynamic, when most of the time these variables can fall back to defaults.  

```ruby
class MyInteractorContext < ActiveInteractor::Context::Base
  attribute :locale, default: -> { I18n.locale }
  attribute :year, default: -> { Date.today.year }
  attribute :format, default: :json
end
```

#### Additional Comments

Following active model practices [active_model/attributes.rb#L25](https://github.com/rails/rails/blob/f33d52c95217212cbacc8d5e44b5a8e3cdc6f5b3/activemodel/lib/active_model/attributes.rb#L25)

Generally being able to define a specific `attribute` can enable more functionality like: **adding `type_cast`**. *Example: Typecasting a checkbox parameter to boolean*

**Comment by aaronmallen** [comment]
@mohameddiaa27 thanks for your feature request.  I am working on implementing the `ActiveModel::Attributes` api in #137 now and hope to have it out in the next patch release.  In the mean time a good work around would be to specify defaults for an attribute in a `before_context_validation` or `before_perform` callback.

**Comment by aaronmallen** [comment]
@mohameddiaa27 the feature has been released and is now available in v1.0.1

**Comment by mohameddiaa27** [comment]
Perfect 👌
Thanks! That was quick 🚀
